cms/djangoapps/contentstore/views/videos.py

```diff
@@ -635,7 +635,11 @@ def videos_index_html(course):
                 'transcript_credentials_handler',
                 unicode(course.id)
             ),
-            'transcription_plans': get_3rd_party_transcription_plans(),
+            'transcript_download_handler_url': reverse_course_url(
```

Comment:
You need to add this to `context` as well at L#625 and update here so that when feature flag is false, this key is still sent to JS and it won't break.